spec/lib/generators/forem/dummy/templates/config/initializers/simple_form.rb

```diff
@@ -0,0 +1,166 @@
+# Use this setup block to configure all options available in SimpleForm.
+SimpleForm.setup do |config|
```

Comment:
Yes, this empty block gives me the same config as without any config block at all.